psi/pmt.go

```diff
@@ -119,7 +119,7 @@ func (p *pmt) parsePMTSection(pmtBytes []byte) error {
 	var elementaryStreams []PmtElementaryStream
 	sectionLength := sectionLength(pmtBytes)
 
-	if len(pmtBytes) < programInfoLengthOffset {
```

Comment:
From what I am seeing in ISO_IEC-13818, Section 2.4.4.8 the total `TS_program_map_section()` up to and including the `program_info_length` is 12 bytes. 

| field | bits |
| --- | --- |
| table_id | 8 |
| section_syntax_indicator | 1 |
| '0' | 1 |
| reserved | 2 |
| section_length | 12 |
| program_number | 16 |
| reserved | 2 |
| version_number | 5 |
| current_next_indicator | 1 |
| section_number | 8 |
|last_section_number | 8 | 
| reserved | 3 |
| PCR_PID | 13 |
| reserved | 4 |
| program_info_length | 12 |


I think this solution does work, it is essentially `len(pmtBytes) < 12` which I totally agree with